src/libs/actions/IOU.js

```diff
@@ -724,11 +721,327 @@ function payIOUReport({
     return promiseWithHandlers;
 }
 
+/**
+ * @param {Object} report
+ * @param {Number} amount
+ * @param {String} currency
+ * @param {String} comment
+ * @param {String} paymentMethodType
+ * @param {String} managerEmail - Email of the person sending the money
+ * @param {Object} recipient - The user receiving the money
+ * @returns {Object}
+ */
+function getSendMoneyParams(report, amount, currency, comment, paymentMethodType, managerEmail, recipient) {
+    const newIOUReportDetails = JSON.stringify({
+        amount,
+        currency,
+        requestorEmail: OptionsListUtils.addSMSDomainIfPhoneNumber(recipient.login),
+        comment,
+        idempotencyKey: Str.guid(),
+    });
+
+    const recipientEmail = OptionsListUtils.addSMSDomainIfPhoneNumber(recipient.login);
+    let chatReport = report.reportID ? report : null;
+    let isNewChat = false;
+    if (!chatReport) {
+        chatReport = ReportUtils.getChatByParticipants([recipientEmail]);
```

Comment:
not sure what this does exactly, `getChatByParticipants` returns a report id the list of participants is the same, but we are sending just one participant, are there reports with just one participant?

Reply:
if you add a comment explaining I'll be happy with this.

Reply:
ok, thanks, that make sense now, happy to resolve this.